vocabularies/geocoding.ttl

```diff
@@ -6,8 +6,9 @@ PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
 PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 <http://opengis.net/ont/geocode/>
-    a owl:Ontology ;
+    a owl:Ontology, skos:ConceptScheme ;

```

Comment:
if it has no members that are skos:Concept and point to it using skos:inScheme, 
then it's not a ConceptScheme